normandy/recipes/tests/test_filters.py

```diff
@@ -39,7 +41,35 @@ def test_uses_only_baseline_capabilities(self, settings):
         if self.should_be_baseline:
             assert capabilities <= settings.BASELINE_CAPABILITIES
         else:
-            assert capabilities > settings.BASELINE_CAPABILITIES
+            assert capabilities - settings.BASELINE_CAPABILITIES
+
+
+class TestVersionFilter(FilterTestsBase):
+    def create_basic_filter(self, versions=None):
+        if versions is None:
+            versions = [72, 74]
+        return VersionFilter.create(versions=versions)
+
+    def test_generates_jexl(self):
+        filter = self.create_basic_filter(versions=[72, 74])
+        assert set(filter.to_jexl().split("||")) == {
+            '(normandy.version>="72"&&normandy.version<"73")',
+            '(normandy.version>="74"&&normandy.version<"75")',
+        }
```

Comment:
The filter already existed, but it was just missing tests so you're adding them here?

Reply:
Yes exactly. Thanks for reading my mind :smile:.